file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs

```diff
@@ -396,22 +412,47 @@ private bool IsDirectoryExist(string directory)
         {
             try
             {
-                _= fileSystem.DirectoryInfo.FromDirectoryName(directory).GetDirectories();
+                _ = fileSystem.DirectoryInfo.FromDirectoryName(directory).GetDirectories();
                 return true;
             }
             catch (Exception)
             {
                 return false;
             }
         }
+
+        private async Task<BatchSearchResponse> SearchBatch()
+        {
+            var filter = buildSearchQuery();
+            var fileShareClient = fileShareClientFactory();
+
+            return await fileShareClient.Search(filter, null, null);
+        }
+
+        private string buildSearchQuery()
+        {
+            var queryBuilder = new StringBuilder();
+
+            queryBuilder.Append($"businessunit eq '{BusinessUnit}'");
+
+            if (Attributes != null)
```

Comment:
Please also check `Attributes` is not empty

Reply:
added empty check as well. Thanks